api/server/server.go

```diff
@@ -42,9 +43,9 @@ type Server struct {
 
 	apiURL string
 
-	specialHandlers []SpecialHandler
 	appListeners    []AppListener
 	middlewares     []Middleware
+	runMiddlewares  []Middleware
```

Comment:
Can Run Middlewares take the `task.Config` as a parameter? I mean if we are middlewaring the runner, we would want to change the config right?

Or is there way to map the context to the task config?